client/my-sites/pages/page/index.js

```diff
@@ -438,34 +420,30 @@ const Page = React.createClass( { // eslint-disable-line react/prefer-es6-class
 					transitionName="updated-trans"
 					transitionEnterTimeout={ 300 }
 					transitionLeaveTimeout={ 300 }>
-					{ this.buildUpdateTemplate() }
+					{ this.props.buildUpdateTemplate() }
 				</ReactCSSTransitionGroup>
 			</CompactCard>
 
 		);
 	},
 
 	updateStatusPublish: function() {
-		this.setState( { showPageActions: false } );
-		this.updatePostStatus( 'publish' );
+		this.props.updatePostStatus( 'publish' );
 		recordEvent( 'Clicked Publish Page' );
 	},
 
 	updateStatusTrash: function() {
-		this.setState( { showPageActions: false } );
-		this.updatePostStatus( 'trash' );
+		this.props.updatePostStatus( 'trash' );
 		recordEvent( 'Clicked Move to Trash' );
 	},
 
 	updateStatusRestore: function() {
-		this.setState( { showPageActions: false } );
-		this.updatePostStatus( 'restore' );
+		this.props.updatePostStatus( 'restore' );
 		recordEvent( 'Clicked Restore' );
 	},
 
 	updateStatusDelete: function() {
-		this.setState( { showPageActions: false } );
-		this.updatePostStatus( 'delete' );
+		this.props.updatePostStatus( 'delete' );
 		recordEvent( 'Clicked Delete Page' );
```

Comment:
If it's not too much trouble can we swap this for
```jsx
const mapDispatchToProps: {
   recordEvent: partial( recordGoogleEvent, 'Pages' ),
//...
}
```

Though I'm fine with deferring this for later if we're trying to keep the diff size down.